app/components/primary-button.tsx

```diff
@@ -0,0 +1,12 @@
+import { Button } from "@/components/ui/button";
+
+interface IButtonInfo {
+    onClick: (e: React.FormEvent) => void;
+    title: string;
```

Comment:
Button の中に入るテキストは title よりも label と呼ばれることが多いですね！
また、React では label で文字列を渡すより、children として ReactNode を渡すのが一般的です
```suggestion
    children: ReactNode
```
こうすれば、以下のようにコンポーネントを使用できます
```TypeScriot
<PrimaryButton>
  Test
</PrimaryButton>
```